dataikuapi/dss/ml.py

```diff
@@ -643,6 +654,386 @@ def __repr__(self):
+
+    @property
+    def expected_min(self):
+        """
+        Returns the min (included) of the expected range or None if it is not defined.
+        Assertion passes if the expected_valid_ratio of rows predicted between expected_min and expected_max (included) is attained.
+
+        :rtype: float
+        """
+        if "expectedMinValue" in self._internal_dict:
+            return self._internal_dict["expectedMinValue"]
+        else:
+            return None
+
+    @expected_min.setter
+    def expected_min(self, expected_min):
+        self._internal_dict["expectedMinValue"] = expected_min
+
+    @property
+    def expected_max(self):
+        """
+        Returns the max (included) of the expected range or None if it is not defined.
+        Assertion passes if the expected_valid_ratio of rows predicted between expected_min and expected_max (included) is attained.
+
+        :rtype: float
+        """
+        if "expectedMaxValue" in self._internal_dict:
+            return self._internal_dict["expectedMaxValue"]
+        else:
+            return None
+
+    @expected_max.setter
+    def expected_max(self, expected_max):
+        self._internal_dict["expectedMaxValue"] = expected_max
+
+
+class DSSMLAssertionsMetrics(object):
+    """
+    Object that represents the assertions metrics for all assertions on a trained model
+    Do not create this object directly, use :meth:`DSSTrainedPredictionModelDetails.get_assertions_metrics()` instead
+    """
+    def __init__(self, data):
+        self._internal_dict = data
+
+    def __repr__(self):
+        return u"{}({})".format(self.__class__.__name__, self.get_raw())
+
+    def get_raw(self):
+        """
+        Gets the raw dictionary of the assertions metrics
+
+        :rtype: dict
+        """
+        return self._internal_dict
+
+    def get_metrics(self, assertion_name):
+        """
+        Retrieves the metrics computed for this trained model for the assertion with the provided name (or None if no
+        assertion with that name exists)
+
+        :param str assertion_name: Name of the assertion
+
+        :returns: an object representing assertion metrics or None if no assertion with that name exists
+        :rtype: :class:`DSSMLAssertionMetrics`
+        """
+        for assertion_metrics_dict in self._internal_dict["perAssertion"]:
+            if assertion_name == assertion_metrics_dict["name"]:
+                return DSSMLAssertionMetrics(assertion_metrics_dict)
+        return None
+
+    @property
+    def passing_assertions_ratio(self):
+        """
+        Returns the ratio of passing assertions
+
+        :rtype: float
+        """
+        return self._internal_dict['passingAssertionsRatio']
+
+
+class DSSMLAssertionMetrics(object):
+    """
+    Object that represents the result of an assertion on a trained model
+    Do not create this object directly, use :meth:`DSSMLAssertionMetrics.get_metrics(self, assertion_name)` instead
```

Comment:
This does not render well when put to the documentation

![image](https://user-images.githubusercontent.com/5690334/102519521-dfdae680-4092-11eb-99e8-da66af5d3025.png)


```suggestion
    Do not create this object directly, use :meth:`DSSMLAssertionMetrics.get_metrics` instead
```